### PR TITLE
Feature: add flake8

### PR DESCRIPTION
**Pull request by Olavhaasie**
Closes PM-135

## Description
Added `flake8` linter

## Changes
* Add `.flake8`
* Remove `roslint`
* Add Travis job
* Fix errors

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    D102,
+    # D103: Missing docstring in public function
+    D103,
+    # D104: Missing docstring in public package
+    D104,
+    # D105: Missing docstring in magic method
+    D105,
+    # D106: Missing docstring in public nested class
+    D106,
+    # D107: Missing docstring in __init__
```
Comment (Joris-JW) [inline .flake8:28]: why add these to the flake8 ignore?

Comment (Olavhaasie) [reply .flake8:28]: They require you to add docstrings to public interfaces. However, that is a lot of docstrings to add and I think it is not always necessary. The name of the function and its parameters could document the method well enough.

**Review by RutgerVanBeek — CHANGES_REQUESTED** [review]
When I first start an RQT input device and then launch the simulation I get an "Input Device connection is from the future" error. One is in seconds from startup or something and the other in seconds since epoch.

**Comment by Olavhaasie** [comment]
> When I first start an RQT input device and then launch the simulation I get an "Input Device connection is from the future" error. One is in seconds from startup or something and the other in seconds since epoch.

This is sadly also an issue in develop. It involves running the simulation with `sim_time`, while the input device is using real time. This could be solved but you could also not do this in the simulation. It will always work on the exo, since that does not uses `sim_time`.

**Review by RutgerVanBeek — APPROVED** [review]

**Pull request merged**